scripts/load_notification_templates.py

```diff
@@ -133,3 +101,73 @@
                 template.save()
 
         logger.info(f"Written template: {template}")
+
+
+def load_sms_templates(offset):
+    logger.info("Writing SMS templates")
+
+    sms_notifications = {
+        PaymentNotificationType.SMS_INVOICE_NOTICE: "invoice_notice_{lang}.txt",
+    }
+
+    for notification_index, (notification_type, template_path) in enumerate(
+        sms_notifications.items()
+    ):
+        template = NotificationTemplate.objects.create(
+            id=notification_index + offset, type=notification_type.value,
+        )
+        for (lang, _name) in languages:
+            with override(lang), switch_language(template, lang):
+                template.subject = notification_type.label
+                full_path = os.path.join(
+                    PROJECT_ROOT, "templates", "sms", template_path
```

Comment:
Where is this directory created? Does it need to be added to generate_email_templates.py?

Reply:
[It was already added](https://github.com/City-of-Helsinki/berth-reservations/blob/c6faa23986b5224b6c29e2214fe9ad37d1f7f6bb/templates/sms/invoice_notice_en.txt), and no need to add it to the script since there's nothing to generate for plain text messages, only load them to the DB.